third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -407,7 +407,15 @@ pub fn bytecode_pipeline(env: &GlobalEnv) -> FunctionTargetPipeline {
     // Reference check is always run, but the processor decides internally
```

Comment:
Move this comment to the `else` branch below (for the old ref safety processor)?